docs/shadow_config_overview.md

```diff
@@ -76,3 +76,72 @@ Acceptable unit *prefixes* are:
 - tebi / Ti
 
 Examples: `20 B`, `100 MB`, `100 megabyte`, `10 kibibytes`, `30 MiB`, `1024 Mbytes`
+
+## YAML Extensions
+
```

Comment:
Mention that this support is experimental, and must be enabled with an experimental flag?

Reply:
Added.